packages/config/config/devices/0x027a/zen17.json

```
@@ -489,6 +489,96 @@
 				}
 			]
 		},
+		{
+			"#": "19",
+			"label": "Reverse the reported values for your selected input type on S1 C terminals",
```

Label is much too long. Something like this would be better:
```suggestion
			"label": "S1 Terminal: Invert Values",
```
also:
* rename param 2 to "S1 Terminal: Input Type"
* Move this directly after param 2
* Copy the option labels from param 2, so they are consistent.